grafonnet/gauge.libsonnet

```diff
@@ -3,6 +3,8 @@
     title,
     datasource=null,
     calc='mean',
+    timeFrom=null,
+    span=null,
```

Comment:
Which version of Grafana are you using?

Since [v5.0](https://community.grafana.com/t/release-notes-v5-0-x/5250), there's been a "new" grid layout. While Grafana attempts to maintain backwards compatibility, I'm not sure `span` works. I've attempted to use it on 5.0.0, 6.6.2, and 6.7.2 with no success.

I'd suggest using something like:

```jsonnet
grafana.gauge.new(
  'Behold, a Gauge',
  timeFrom='17m',
) {
  gridPos: { w: 24 },
},
```

`gridPos` should work for all panels.

Reply:
v6.3.2. Span seems to work?

Reply:
Weird, I'm having trouble getting it to work, but I believe you. And it's used in a few other components so let's roll with it.